app/pulp/app/serializers/repository.py

```diff
@@ -15,11 +15,26 @@ class RepositorySerializer(ModelSerializer):
         help_text='A unique name for this repository.'
     )
 
+    description = serializers.CharField(
+        help_text='An optional description.',
+        required=False
+    )
+
+    last_content_added = serializers.DateTimeField(
+        help_text='When content was last added.',
```

Comment:
Could this help text be more verbose, like "Timestamp of the most recent addition of content to this repository." ?

Same for `last_content_removed`.